src/pages/signin/SubmitButton.js

```diff
@@ -3,80 +3,63 @@ import PropTypes from 'prop-types';
 import {
     ActivityIndicator, Text, TouchableOpacity, View
 } from 'react-native';
-import {withOnyx} from 'react-native-onyx';
 import styles from '../../styles/styles';
 import themeColors from '../../styles/themes/default';
-import ONYXKEYS from '../../ONYXKEYS';
 import {restartSignin} from '../../libs/actions/Session';
 
 const propTypes = {
     // The text for the button label
     text: PropTypes.string.isRequired,
 
     // Indicates whether the button should be disabled and in the loading state
-    isLoading: PropTypes.bool.isRequired,
+    isLoading: PropTypes.bool,
 
     // A function that is called when the button is clicked on
     onClick: PropTypes.func.isRequired,
 
     // Whether or not to show the restart sign in button
     showRestartButton: PropTypes.bool,
-
-    /* Onyx Props */
-
-    // The session of the logged in person
-    session: PropTypes.shape({
-        // Error to display when there is a session error returned
-        error: PropTypes.string,
-    }),
 };
 const defaultProps = {
     showRestartButton: true,
-    session: {},
+    isLoading: false,
 };
 
-const SubmitButton = (props) => {
-    // When there is an error in the session (a sign on error) then the button should be
-    // enabled so the form can be submitted again
-    const isLoading = props.isLoading && !props.session.error;
-    return (
-        <>
+const SubmitButton = props => (
```

Comment:
Suggestion to rename this to something like `SignInControls` or `LoginButtonGroup` a "submit button" sounds more general purpose and the presence of "Change Expensify login" tells me this is more specific than the name would suggest.

Reply:
Ah, that's a great point. I had a similar thought when this component was tied to the `session` property in Onyx. I would like to move the "change expensify login" to be outside the button control, and that will allow this to be more generic and maybe move it to the components folder.